OpenRA.Platforms.Default/ThreadedGraphicsContext.cs

```diff
@@ -155,6 +165,22 @@ internal void ReturnVertices(Vertex[] vertices)
 					verticesPool.Push(vertices);
 		}
 
+		internal uint[] GetIndices(int size)
```

Comment:
Calling code creates verticies and vertex buffers using `TempBufferSize` == `BatchSize` and does so often - so object pooling of verticies benefits those code paths.

Since nothing creates indicies each frame at the moment, I would leave out all the object pooling for now until we need it. When we do need it it might need to work on something other than `BatchSize` to work well.

Reply:
I don't think a custom implementation is worth it given that we'll probably rip it out soon

Reply:
Sure, so can we remove the current pooling?

Reply:
What would be the desired system? This is the exact copy-paste of the vertex code. Should we instead of pooling use Array.Resize? Or perhaps should we should have a `SetStaticData`?

Reply:
For now we can do blocking `device.Send` with the callers passed in array. This means we don't need to worry about array copies or pooling.

Reply:
removed poling. Given that we only render quads (and will only render quads)) I've just made the buffer pre-generate quad indices and be static